scripts/odemis-start-turret.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Starts the SPARC spectrograph and select a specific turret
+# To start automatically with Odemis, run with:
+# sh -c "env PYTHONPATH=$HOME/development/odemis/src/ $HOME/development/odemis/scripts/odemis-start-turret.py 1 && odemis-start"
```

Comment:
to follow the standardized docstrings; could you  add a docstring with creation date, author etc?